packages/api/tests/routes.test.js

```diff
@@ -0,0 +1,13 @@
+const request = require('supertest')
+const app = require('../server')
+describe('GET Endpoints', () => {
+  it('should create a new GET request', async () => {
+    const res = await request(app)
+      .get('/greeting')
+      .send({
+        userId: 1,
+        title: 'test is cool',
+      })
```

Comment:
i know this is a dummy endpoint and test, but technically this `.send()` is not needed and can be removed (it's using GET params i think)